### PR TITLE
Package upgrades

### PR DESCRIPTION
**Pull request by MatthewFrench**
Upgraded the command line.
Made scripts no longer load bin and obj folders because it causes undefined behavior. (I'm so bad at single purpose pull requests)

### CODE REVIEW

File: GameServerApp/Program.cs [hunk 1]
```diff
@@ -19,70 +19,70 @@ private static void Main(string[] args)
...
-                Encoding.UTF8.GetString(Resources.GameServerSettings)));
-            if (configGameServerSettings.AutoStartClient)
-            {
-                var leaguePath = configGameServerSettings.ClientLocation;
-                if (Directory.Exists(leaguePath))
-                {
-                    leaguePath = Path.Combine(leaguePath, "League of Legends.exe");
-                }
-                if (File.Exists(leaguePath))
+            ArgsOptions.Parse(args).WithParsed((parsedArgs) => {
```
Comment (MythicManiac) [inline GameServerApp/Program.cs]: Just cast this to a variable if possible, higher indentation level is generally a bad practice

Comment (MythicManiac) [reply GameServerApp/Program.cs]: Not to mention it changes the entire diff even though technically you modified just a single line

File: GameServerLib/GameServerLib.csproj [hunk 2]
```diff
@@ -57,56 +56,49 @@
...
-      <Private>True</Private>
     </Reference>
     <Reference Include="Microsoft.Win32.Primitives, Version=4.0.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
       <HintPath>..\packages\Microsoft.Win32.Primitives.4.3.0\lib\net46\Microsoft.Win32.Primitives.dll</HintPath>
     </Reference>
     <Reference Include="Newtonsoft.Json, Version=9.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
       <HintPath>..\packages\Newtonsoft.Json.9.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
-      <Private>True</Private>
+    </Reference>
+    <Reference Include="Ninject, Version=3.2.0.0, Culture=neutral, PublicKeyToken=c7192dc5380945e7, processorArchitecture=MSIL">
```
Comment (MythicManiac) [inline GameServerLib/GameServerLib.csproj]: Shouldn't Ninject be gone?

File: GameServerLibTests/GameServerLibTests.csproj [hunk 3]
```diff
@@ -47,6 +47,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\lib\Microsoft.VisualStudio.QualityTools.UnitTestFramework.dll</HintPath>
     </Reference>
+    <Reference Include="Ninject, Version=3.2.0.0, Culture=neutral, PublicKeyToken=c7192dc5380945e7, processorArchitecture=MSIL">
```
Comment (MythicManiac) [inline GameServerLibTests/GameServerLibTests.csproj]: same here

**Review by MythicManiac — COMMENTED** [review]
Pointed out some issues. I'm also wondering about all the added references, if they're used why were they added only now? What caused the addition? It's not a big issue but I'm just wondering.

**Comment by MythicManiac** [comment]
Nice work tho 👍

File: GameServerApp/Program.cs (hunk 1 above)
Comment (MatthewFrench) [reply GameServerApp/Program.cs]: Fixed

File: GameServerLib/GameServerLib.csproj (hunk 2 above)
Comment (MatthewFrench) [reply GameServerLib/GameServerLib.csproj]: Now removed

File: GameServerLibTests/GameServerLibTests.csproj (hunk 3 above)
Comment (MatthewFrench) [reply GameServerLibTests/GameServerLibTests.csproj]: Removed

**Comment by MatthewFrench** [comment]
I'm not sure why some of the system references were added. I'm assuming it's due to new changes in Visual Studio, project format and the libraries themselves.
(I totally snuck in a one line fix for Log4Net's complaining)

**Review by MythicManiac — APPROVED** [review]

**Pull request merged**